ci/docker/conda.dockerfile

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 ARG arch=amd64
-FROM ${arch}/ubuntu:18.04
+FROM ${arch}/ubuntu:22.04
```

Comment:
I don't mind using 22 but is there a reason we skip 20?

Reply:
No.
It's just no-reason to use Ubuntu 20.04. :-)